CDCgov/prime-reportstream
# [Bug]: Site Wide: Navigating through the application buttons does not…

… scroll you to the top of destination page

This PR adds a ScrollToTop component as a sibling to App (not parent of, so as not to cause unnecessary re-renders) that will forcefully set the window to scroll back to the top on page changes.

Test Steps:
1. Login as admin, go to Admin->Organization Settings
2. Choose the "waters" org from the list
3. Scroll down to a Receiver setting and click Edit or Clone
4. Observe that you are NOT halfway down the page

## Changes
- Created ScrollToTop component
- Added it as sibling to App

## Checklist

### Testing

### Process

## Linked Issues
- Fixes #4327
